federation/merge_schemas.go

```diff
@@ -312,6 +338,36 @@ func mergeEnumValues(a, b []introspectionEnumValue, mode MergeMode) ([]introspec
 	return merged, nil
 }
 
+func mergeDirectives(a, b []introspectiveDirective, mode MergeMode) ([]introspectiveDirective, error) {
```

Comment:
Unlike the types that can be different in gql servers, directive is a global feature that should be supported by the merged schema. 
Our directive introspection queries are hardcoded pre federation. I think it is reasonable to also hardcode them in the merged introspection schema.

Reply:
Is the implication here that the directives will be the same for all federated schemas?

Reply:
I think so, directives is a global feature.